lib/ansible/plugins/httpapi/fortios.py

```diff
@@ -33,7 +33,7 @@
 DOCUMENTATION = """
 ---
 author:
-    - Miguel Angel Muñoz (@magonzalez)
```

Comment:
You don't necessarily have to do this, but you do need to tell Python what the file's encoding is if it is not ASCII

See https://www.python.org/dev/peps/pep-0263/ for how to do that